.github/workflows/container-release.yml

```diff
@@ -4,9 +4,18 @@ on:
   push:
     branches:
       - "master"
-  schedule:
-    - cron: 0 0 * * *
-
+    paths-ignore:
+      - "*.md"
+      - LICENCE
+      - TRANSLATION
+      - invidious.service
+      - .git*
```

Comment:
Isn't this including the .github folder too? What happen if we change something in it which is related to how the docker image is built, does it skip the compilation or not when the changes are pushed?

Reply:
No. The `paths-ignore` only includes files, hence all of the globbing in the directory paths. If you take a look at my previous workflow pull requests, the jobs ran during checks is always from the updated workflow. `.git*` *shouldn't* prevent the workflow files from being ran.